applications/crossbar/src/modules/cb_groups.erl

```diff
@@ -154,7 +154,7 @@ delete(Context, _) ->
 -spec create(cb_context:context()) -> cb_context:context().
 create(Context) ->
     OnSuccess = fun(C) -> on_successful_validation('undefined', C) end,
-    cb_context:validate_request_data(<<"groups">>, Context, OnSuccess).
+    cb_context:validate_request_data(<<"callflows.ring_group">>, Context, OnSuccess).
```

Comment:
Weird. This file with same name & id also exists in 3.22 (`"groups"` schema is also queried there).
Maybe that's not the right schema?

@lazedo @jamesaimonetti @k-anderson